sample_data_OeREBKRMtrsfr_V2_0/themes.json

```diff
@@ -0,0 +1,134 @@
+[
```

Comment:
Why do we need a special themes json here? Wouldnt this be something for a dev.themes in the sample_data folder?